website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -580,25 +581,39 @@ version: 1.0.0
 ...
 
 models:
-  +snowflake_warehouse: "EXTRA_SMALL"    # use the `EXTRA_SMALL` warehouse for all models in the project...
+  +snowflake_warehouse: "EXTRA_SMALL"    # default snowflake virtual warehouse for all models in the project.
```

Comment:
I think branding should override any historical casing we may do (but also why don't we sentence case these?)
```suggestion
  +snowflake_warehouse: "EXTRA_SMALL"    # default Snowflake virtual warehouse for all models in the project.
```